examples/ee-demo/app.js

```diff
@@ -58,7 +58,9 @@ export default class App extends React.Component {
     this.forceUpdate();
     await this.eeApi.initialize(EE_CLIENT_ID); // Client id to your EE application
 
-    const eeImage = ee.Image('CGIAR/SRTM90_V4');
+    // const eeImage = ee.Image('CGIAR/SRTM90_V4');
+    const eeImage =
```

Comment:
Can we generate this by serializing the ee.Image instead of hard coding the serialization?

Reply:
Yes, the reason I hard coded it was to test passing the token to the layer, since `ee.Image` fails if authentication hasn't been called yet.

Since the app always starts with Google auth, it's fine to use `ee.Image().serialize()`.

Should there be a separate demo for using token auth in JS? If the user has already authenticated, say with the Python API/CLI, they could pass the refresh token to the JS layer and the JS layer deals with loading the access token. 

What do you think about doing this in JS rather than Python? Right now in #18 that assumes the prop is an _access token_ and not a _refresh token_. If we allow the prop to be a _refresh token_, we can handle generating the access token on the fly from the JS side, and not need to reimplement it in Python.

Reply:
After all, if you have a valid `refresh_token`, it's simple to get an access token with one request to `https://www.googleapis.com/oauth2/v4/token`.

Reply:
Yes that is always a difficult question. I have duplicating an entire demo for just one little difference. 

That said, authentication is by far going to be the biggest issue for users. 

Alternative ideas:
- Maybe we could read a `token` or `clientId` parameter from the URL.


We should definitely have a really good "Authenticating with EE" doc in the user guide section of the website. 

Feel free to work on docs in parallel with code...